### DISCUSSION TITLE
Enhanced delegation options, add delegate history, and streamlined UI: Empowering choices, minimizing clutter

### DISCUSSION THREAD

**Discussion by Juanma0x** [jeluard/swipealot#481] [General]
I hope it's not too late to chime in, but I have some suggestions:

**Delegation options**
- It would be great to have a network selector, including Polkadot and Kusama.
- I would like to be able to choose my conviction voting, even if the default is "no conviction."
- It would be helpful to view the voting history of each delegate.
- Display information about the number of tokens/accounts delegated to each delegate, offering insights into their level of support. [Related](https://github.com/subsquid/bounties/issues/9).
- [Edited] Do not have to delegate all your transferable funds.

**UI improvements**
- Replace horizontal scrolling for delegates in the top section with a paginated table, providing a more organized and easily navigable interface.
- The button to delegate on all tracks within each delegate's card should be vertically aligned with the other buttons, regardless of the amount of information provided by the delegate.
- Tracks with no active referenda could be made smaller since they don't contain much information.
- Tracks with active referenda could be collapsible. To differentiate them from non-active tracks, you could use a different color or incorporate an emoji/icon.
- Include direct links to the OpenGov wiki, allowing users to access additional information about the platform without cluttering the main interface.

**Comment by jeluard** [comment]
Thanks for chiming in! We will definitely consider those points for our future iterations!
